client/pages/index.tsx

```diff
@@ -31,7 +31,12 @@ const Dashboard: NextPage = () => {
       <div className='flex-1 flex flex-col min-h-0'>
         <div className='flex justify-center'>
           <Button variant='outline'>Cancel</Button>
-          <Button variant='outline' type='button' disabled={isFirstStep} onClick={handleClick}>
+          <Button
+            variant='outline'
+            type='button'
+            disabled={isFirstStep}
+            onClick={() => handleClick('')}
```

Comment:
was this giving you a type error because of the removal of the `props: any` bit?

Reply:
Ahh because of the parameter on `handleClick` probably

Reply:
Yeah, exactly! Had to either do it like this or add props: any or whatever to the Button onClick. If there's a better way - let me know!